test/mocha.opts

```diff
@@ -1,6 +1,8 @@
 --require ts-node/register
 --ui mocha-typescript
 --require source-map-support/register
+#Increasing test timeout for node v14, refer To: https://github.com/amzn/ion-js/issues/581
```

Comment:
```suggestion
# Increasing test timeout to accommodate slow tests that break the CI build
# Refer to: https://github.com/amzn/ion-js/issues/581
```

`v14` may have been the one to fail during that particular build, but I've seen the others fail sporadically too.